Description: Clean up the TCPUDPFlow.bro script
 
 securityonion-tcpudpflow (001-0ubuntu0securityonion1) precise; urgency=low
 .
   * Script updates.
Author: Scott Allen Runnels <srunnels@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- securityonion-tcpudpflow-001.orig/TCPUDPFlow.bro
+++ securityonion-tcpudpflow-001/TCPUDPFlow.bro
@@ -10,13 +10,12 @@ redef udp_content_deliver_all_orig = T &
 
 event http_header(c: connection, is_orig: bool, name: string, value: string)
 	{
-    print fmt("DEBUG: %s: %s", name, value);
+    print fmt("%s: %s", name, value);
 	}
 
 event http_reply(c: connection, version: string, code: count, reason: string)
 	{
-    print fmt("%s.%d-%s.%d:", c$id$resp_h, c$id$resp_p, c$id$orig_h, c$id$orig_p);
-    print fmt("%s - %s", code, reason);
+    print fmt("%s.%d-%s.%d: %s %s", c$id$resp_h, c$id$resp_p, c$id$orig_h, c$id$orig_p, code, reason);
 	}
 
 event http_entity_data(c: connection, is_orig: bool, length: count, data: string)
@@ -26,8 +25,7 @@ event http_entity_data(c: connection, is
 
 event http_request(c: connection, method: string, original_URI: string, unescaped_URI: string, version: string)
 	{
-    print fmt("%s.%d-%s.%d:", c$id$orig_h, c$id$orig_p, c$id$resp_h, c$id$resp_p);
-    print fmt("%s %s", method, original_URI);
+    print fmt("%s.%d-%s.%d: %s %s", c$id$orig_h, c$id$orig_p, c$id$resp_h, c$id$resp_p, method, original_URI);
 	}
 
  event connection_state_remove(c: connection)
@@ -37,10 +35,6 @@ event http_request(c: connection, method
          print c$dns;
  	}
 
-event bro_init()
-    {
-    }
-
 event udp_contents(u: connection, is_orig: bool, contents: string)
 	{
     if (is_orig)
